ecosystem/sep-0018.md

```diff
@@ -83,10 +100,3 @@ ensuring namespaces are not given values. Example:
 profile.language.language = en
 profile.language.region = US
```

Comment:
I think we should change this example to use the `app.example_org` namespace so that profile doesn't exist in this document.